sources/Core/Numerics/Matrix3x3.cs

```diff
@@ -111,5 +121,15 @@ public string ToString(string? format, IFormatProvider? formatProvider)
                 .Append('>')
                 .ToString();
         }
+
+        /// <summary>Creates a transposed version of this <see cref="Matrix3x3" />.</summary>
+        /// <returns>The transposed <see cref="Matrix3x3" />.</returns>
+        public static Matrix3x3 Transpose(Matrix3x3 m)
+        {
+            return new Matrix3x3(
+                new Vector3(m.X.X, m.Y.X, m.Z.X),
+                new Vector3(m.X.Y, m.Y.Y, m.Z.Y),
+                new Vector3(m.X.Z, m.Y.Z, m.Z.Z));
```

Comment:
nit: When formatting to multiple lines like this, I'd prefer the closing brace to be on its own line. I find it helps to visually parse the end
```suggestion
                new Vector3(m.X.Z, m.Y.Z, m.Z.Z)
            );
```

Reply:
ok. I applied it to all modified files.